src/fireblocks-sdk.ts

```diff
@@ -22,6 +22,8 @@ import {
     InternalWalletAsset,
     MaxSpendableAmountResponse,
     MaxBip44IndexUsedResponse,
+    PaginatedAddressesResponse,
```

Comment:
what about AddressResponse ?

Reply:
i can import it but there are not use in it,
i only use PaginatedAddressesResponse as the return value of the function:
Promise<PaginatedAddressesResponse>
and that contain the AddressResponse as an array

Reply:
Ok, if that's not a must -leave it as is.